lib/receptor_controller/client/directive_non_blocking.rb

```diff
@@ -48,6 +48,7 @@ def call(body = default_body)
 
         # registers message id for kafka responses
         response_worker.register_message(msg_id, self)
+        logger.debug("Receptor response: registering message #{msg_id}".tap { |msg| msg << " req: #{body[:payload]}" if ENV["LOG_ALL_RECEPTOR_MESSAGES"]&.to_i != 0 })
```

Comment:
@lindgrenj6 Instead of the payload can we log just the href_slug all the time for a debug message irrespective of the LOG_ALL_RECEPTOR_MESSAGES being set.